doc/conf.py

```diff
@@ -72,3 +72,20 @@
     'docs': ('https://docs.nextstrain.org/en/latest/', None),
 }
 
+
+# -- Linkchecking ------------------------------------------------------------
+
+linkcheck_ignore = [
+     # we have links to localhost for explanatory purposes; obviously
+     # they will never work in the linkchecker
+     r'http://127.0.0.1:\d+',
+     r'http://localhost:\d+',
+]
+linkcheck_anchors_ignore_for_url = [
+     # Github uses anchor-looking links for highlighting lines but
+     # handles the actual resolution with Javascript, so skip anchor
+     # checks for Github URLs:
+     r'https://github.com',
+     r'https://console.aws.amazon.com/batch/home',
+     r'https://console.aws.amazon.com/ec2/v2/home',
+]
```

Comment:
These patterns are all regexes, and so should be 1) anchored (e.g. to the start, `^`) and 2) have metachars, like `.` escaped (e.g. either `\.` or `[.]`).

Reply:
> These patterns are all regexes, and so should be 1) anchored (e.g. to the start, `^`) and 2) have metachars, like `.` escaped (e.g. either `\.` or `[.]`).

I didn't anchor them at the front because I read the underlying code and it's using `re.match()`, not `re.search()`. I'll escape the metachars (and add the anchors).